src/app/Shared/Services/Api.service.tsx

```diff
@@ -1450,7 +1452,12 @@ export class ApiService {
       } else {
         Promise.resolve(error.xmlHttpResponse.body as string).then((detail) => {
           if (!suppressNotifications) {
-            this.notifications.danger(`Request failed (${error.xmlHttpResponse.status} ${error.message})`, detail);
+            try {
+              const body = JSON.parse(detail).data.reason;
+              this.notifications.danger(`${title}`, body);
```

Comment:
I think ```${title}``` can just be `title` - pass the string variable directly instead of interpolating it inside of another string.